library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -0,0 +1,415 @@
+        }
+    }
+
+    #site-rubrics-title {
+        display: flex;
+        justify-content: space-between;
+        div {
+            font-weight: bold;
+            flex: 1;
+        }
+    }
+
+    #shared-rubrics-title {
+        display: flex;
+        justify-content: space-between;
+        div {
+            font-weight: bold;
+            flex: 1;
+        }
+    }
+
+    iron-icon:not(.collpase-icon):not(.reorder-icon):not(.comment-icon) {
+        background: #6e757d;
+        color: #f2f2f4;
+        border-radius: 50px;
+        padding: 5px;
+        cursor: pointer;
+        height: 20px;
+        width: 20px;
+        &:hover {
+            background: #e7e7e7;
+            color: #6e757d;
+        }
+        &.add {
+            color: #ffffff;
+            background: #0093b8;
+        }
+    }
+
+    iron-icon.sideways {
+        transform: rotate(90deg);
+    }
+
+    iron-icon.comment-icon {
+        height: 15px;
+        width: 15px;
+        margin: 5px auto;
+        display: block;
+        &.active {
+            fill: orange;
+        }
+    }
+
+    .btn.add-rubric {
+        color: #0093b8;
+    }
+
+    .rubric-details {
+        padding: 10px;
+    }
+}
+
+/** External to the tool **/
+.sakai-rubric-association {
+    .disabled {
+        label {
+            color: #ccc;
+        }
+    }
+    .rubrics-list {
+        padding-left: 20px;
+    }
+    select.form-control {
+        -webkit-appearance: menulist;
+        width: auto;
+        display: inline-block;
+        margin: 0 20px 10px 0;
+        #rubricslist {
+            display: inline-block;
+        }
+    }
+}
+
+.rubrics-lightbox {
+    position: absolute;
+    z-index: 999;
+    background: rgba(0,0,0,0.5);
+
+    sakai-rubric-criterion-preview {
+        display: block;
+        padding-bottom: 20px;
+    }
+    .container {
+        background: #fff;
+        margin: 80px auto;
+        position: relative;
+        > a {
+            text-decoration: none;
+            padding: 1px 7px 6px;
+            font-weight: bold;
+            font-size: 18px;
+            background: #fff;
+            border: 1px solid #555;
+            border-radius: 30px;
+            display: block;
+            line-height: 1;
+            position: absolute;
+            top: -10px;
+            right: -10px;
+        }
+    }
+}
+
+.criterion {
+    &.sakai-rubric-criterion,
+    &.sakai-rubric-criterion-preview,
+    &.sakai-rubric-criterion-grading,
+    &.sakai-rubric-criterion-student,
+    &.sakai-rubric-criterion-readonly {
+        width: 100%;
+        border-bottom: 1px solid #d3d3d3;
+        .criterion-row {
+            width: 100%;
+            height: 100%;
+            display: table;
+            border: 1px solid #d3d3d3;
+            border-width: 1px 0 0 0;
+            &.sortable-ghost, .sortable-ghost {
+                opacity: 0.25;
+            }
+            &.sortable-drag, .sortable-drag {
+                cursor: grabbing;
```

Comment:
I prefer to put the non-browser-prefixed property last so the certified property gets applied (last) as browsers adopt the new property going forward.